fedora-prime-select

```diff
@@ -1,61 +1,192 @@
 #!/usr/bin/env bash
 set -e
 
```

Comment:
the simpliest way is to ask directly for su password than to redo the cmd line with sudo.